ansible-roles/ansible-role-springboot_foundation/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: "Deploy Springboot foundation"
+  include_tasks: core.yml
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version == "7" or ansible_distribution_major_version == "8"  or ansible_distribution_major_version == "9"
```

Comment:
Why put things for Fedora in galaxy file, ... and not authorize it here?

Reply:
Was a bit lazy to implement the Fedora tests, but will...